lib/alephant/broker/load_strategy/revalidate/strategy.rb

```diff
@@ -0,0 +1,56 @@
+require "alephant/broker/cache"
+require "alephant/broker/errors"
+require "alephant/logger"
+require "faraday"
```

Comment:
I like the idea of being explicit with the `require` calls within namespaces. So in this case, we're referencing Fetcher and Refresher, so I think I'd prefer these to be explicitly referenced within this file so it's clear where these dependencies can be found